tests/unit/s2n_protocol_preferences_test.c

```diff
@@ -29,6 +31,7 @@ int main(int argc, char **argv)
     const size_t protocol1_len = strlen((const char *)protocol1);
     const uint8_t protocol2[] = "protocol abc 2";
     const size_t protocol2_len = strlen((const char *)protocol2);
+    uint8_t protocol3[] = "protocol3";
```

Comment:
Why not keep this a `const` as well, I don't see it modified in the test. Also nit: we could use `protocol abc 3` as it's value to maintain the pattern.

Reply:
It's not modified in the test, but s2n_blob doesn't accept const pointers and it is used in some blobs. And why match "protocol abc 2" instead of "protocol1"? I believe "protocol abc 2" was probably specifically intended to be a different size than protocol1, and we have no such restriction for protocol3.